regression-tests/test-results/gcc-13-c++2b/mixed-bugfix-for-ufcs-non-local.cpp.output

```diff
@@ -1,41 +1,41 @@
 In file included from mixed-bugfix-for-ufcs-non-local.cpp:6:
 ../../../include/cpp2util.h:2100:1: error: lambda-expression in template parameter type
- 2100 |         return value(x);
+ 2100 | }
       | ^
 ../../../include/cpp2util.h:2137:59: note: in expansion of macro ‘CPP2_UFCS_’
- 2137 |     return false;
-      |                                                           ^         
+ 2137 | template<typename T, specialization_of_template<std::optional> X>
+      |                                                           ^~~~~~~   
```

Comment:
Some GCC 10 version doesn't output these `~` squiggles.
Maybe @hsutter's local version or a CI runner's.
In the `main` branch's commit c5feb428bd1010b00df708236eeb7241d4779883,
they are again removed: <https://github.com/hsutter/cppfront/commit/c5feb428bd1010b00df708236eeb7241d4779883#diff-aa97a613d9dea020a78ec4046393f329f881475e3d67f8796e6e8afec5849968R4>.